src/abstractdataframe/abstractdataframe.jl

```diff
@@ -970,15 +970,34 @@ julia> dropmissing(df, [:x, :y])
 @inline function dropmissing(df::AbstractDataFrame,
                              cols::Union{ColumnIndex, MultiColumnIndex}=:;
                              view::Bool=false, disallowmissing::Bool=!view)
+    # Identify Bool mask of which rows have no Missings
     rowidxs = completecases(df, cols)
     if view
         if disallowmissing
             throw(ArgumentError("disallowmissing=true is incompatible with view=true"))
         end
         return Base.view(df, rowidxs, :)
     else
-        newdf = df[rowidxs, :]
-        disallowmissing && disallowmissing!(newdf, cols)
+        # Faster when there are many columns (indexing with integers than via Bool mask)
+        # or when there are many Missing (as we skip a lot of iterations)
```

Comment:
```suggestion
        # or when there are few Missing (as we skip a lot of iterations)
```

Reply:
I’m mot sure I follow your edit here, @bkamins 

My intuition is as follows:

Let’s take a vector of length 1000.
If we had no missings, findall returns 1:000 and we need to copy 1000 elements in the loop to the new vector.
If we had 999 missings, findall returns 1:1 and we are done after one loop iteration.
The above wouldn’t be the case for a Boolean mask, where we would need to check each one of the 1000 elements.

So shouldn’t the comment be reverted to previous form? ie, the more missings we have, the more iterations we can skip when copying the data?

Reply:
You are right - I am not sure why I was confused (but it is good that we agree - I had the same in my mind 😄)